src/components/home/Landing.jsx

```diff
@@ -3,7 +3,7 @@ import landing from "@/public/images/landing.webp";
 
 const Landing = () => {
   return (
-    <div className="p-6 pb-10  border-8 flex justify-center items-center border-biscuits-orange-200 rounded-3xl bg-biscuits-peach-200 bg-opacity-60 w-3/4">
+    <div className=" p-1 md:p-6 pb-3 md:pb-10 border-4 md:border-8 flex justify-center items-center border-biscuits-orange-200 rounded-3xl bg-biscuits-peach-200 bg-opacity-60 w-3/4">
       <Image src={landing} alt="landing" width={1250} height={1250} />
```

Comment:
is it possible to scale up the image on mobile?

Reply:
Padding capped the image size for mobile, so when I made the padding responsive the image size on mobile got bigger. I can make the image even more larger, but I think it might go out of the box because right now it is touching the corners.
![image](https://github.com/user-attachments/assets/9154f57f-f5fa-4a7d-843b-0fdf356b2b8d)

Reply:
![image](https://github.com/user-attachments/assets/4f12c3c8-25e0-4fe0-8815-4ff7a6721186)
this is how it was before i made the padding responsive

Reply:
okii i think before looks fine so this should be good to go